pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.isort]
+profile = "black"
```

Comment:
I would avoid `pyproject.toml` because we're not a python package.

Perhaps the[ `.isort.cfg` option works](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#custom-configuration). It would also be fine by me to just remove `isort` from the pre-commit config file.